lib/postgresql.js

```diff
@@ -15,6 +15,11 @@ var ParameterizedSQL = SqlConnector.ParameterizedSQL;
 var util = require('util');
 var debug = require('debug')('loopback:connector:postgresql');
 
+// Add support for node 0.10.x
+if (typeof Promise == 'undefined') {
+  global.Promise = require('promise-polyfill');
```

Comment:
This is actually weird.

https://github.com/strongloop/loopback-datasource-juggler/blob/master/3.0-RELEASE-NOTES.md#always-use-bluebird-as-promise-library
> In version 3.0, we always use bluebird as our promise library instead of `global.Promise`. We consider Bluebird API a part of LoopBack API from now on, you are welcome to use any Bluebird-specific methods in your applications.

When using juggler@3, it should not be needed to patch `global.Promise`. Unless there is a rouge test in juggler that uses `global.Promise` instead of `require('bluebird')`, in which case we need to make the fix in juggler.

@superkhau @0candy could you PTAL?